app/components/markdown/markdown.js

```diff
@@ -222,11 +191,17 @@ export default class Markdown extends PureComponent {
     }
 
     renderHeading = ({children, level}) => {
-        const style = getStyleSheet(this.props.theme);
-
+        let style = getStyleSheet(this.props.theme);
+        style = {
```

Comment:
How does this even work? Isn't `style.block` a number because style is a StyleSheet? Even if it does work, is this doing anything different than the old version?

Also, I don't like how we're changing the type of `style` from a StyleSheet to a plain object. If we're changing the data type of a variable, we should give it a new name like this could be `containerStyle` or `viewStyle`